ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -138,6 +138,10 @@ def cos(self, name=None):
     def exp(self, name=None):
         return ivy.exp(self._ivy_array)
 
+    @with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")

```

Comment:
Is there any reference to these dtypes being unsupported?

Reply:
these are supported here you can see in the paddle docs
![paramet](https://github.com/unifyai/ivy/assets/49721249/f0727139-51e6-4b75-be8f-803770043194)